server/audit/views.py

```diff
@@ -383,3 +385,36 @@ def check_item(self, request):
                 status=status.HTTP_400_BAD_REQUEST)
 
         return response
+
+
+class RecommendationViewSet(LoggingViewset):
+    """
+    Allows obtaining all clients and updating them
+    """
+    http_method_names = ['get']
+    serializer_class = GetBinToSKSerializer
```

Comment:
Is this the correct serializer for this viewset ? I see you added RecommendationBinSerializer but havent used it anywhere.

Reply:
Yes this is the one used I'm going to delete the other serializer